### PR TITLE
Feature#5/symptoms UI

### LINKED ISSUE #5: Symptoms From Screen UI
# REQUIREMENTS
-Implement the UI design.
-Implement the form flow with Formik.

# DESIGN
Provide references to images of the design, how it looks like now, and how it should look like.
Captura de Pantalla 2021-03-19 a la(s) 13 00 57
# Acceptance Criteria
List all required acceptance criteria
- Good code quality
- As close as posible to the design

### PR DESCRIPTION
**Pull request by Joaquin-Niembro**
# Description
What did you implemented/Why did you implemented this:
-We implemented the UI of the screen.
-Implemented the form functionality.
 
 # Testing
-git clone repo
-git switch Feature#5/symptoms UI
-npm install
-npm run start
-npm run android
 

  
  ## Screenshots 
image

### CODE REVIEW

**Review by fernandoamz — APPROVED** [review]
LGTM!

**Review by mariam-rivera — APPROVED** [review]

**Review by jjcr11 — APPROVED** [review]

File: AlertaCovid/src/navigations/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export * from './AppNavigator'
+export * from './AuthNavigator'
+
+
```
Comment (LeyaimMagma) [inline AlertaCovid/src/navigations/index.js:4]: Remove the extra space

File: AlertaCovid/src/screens/Symptoms.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      <Image
+        source={require('../assests/images/virusverde.png')}
+        style={SymptomsStyles.virus3}
+      />
+      <Title />
+      <Form symptoms={symptoms} setSymptoms={setSymptoms} />
+      <Submit symptoms={symptoms} />
+    </View>
+  );
+};
```
Comment (LeyaimMagma) [inline AlertaCovid/src/screens/Symptoms.js:49]: can you refactor this?

File: AlertaCovid/src/utils/functions/symptomsFuncs.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const filterSymptoms = (symptoms, id, newValue, title) => {
+  let filteredSymptoms = symptoms.filter((symptom) => symptom.id !== id);
+  return [...filteredSymptoms, {id, has: newValue, title}];
+};
+
+export {filterSymptoms};
+
+
```
Comment (LeyaimMagma) [inline AlertaCovid/src/utils/functions/symptomsFuncs.js:8]: Remove the extra space here

**Pull request merged**